web/src/main/java/com/epam/esm/web/controller/TagController.java

```diff
@@ -0,0 +1,36 @@
+package com.epam.esm.web.controller;
+
+import com.epam.esm.service.dto.TagDto;
+import com.epam.esm.service.exception.ValidationException;
+import com.epam.esm.service.service.TagService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/tags")
+public class TagController {
+
+    private final TagService tagService;
+
+    public TagController(TagService tagService) {
+        this.tagService = tagService;
+
+    }
+
+    @PostMapping
+    public void saveTag(@RequestBody TagDto tag) throws ValidationException {
+        tagService.saveTag(tag);
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<TagDto> getTagById(@PathVariable Long id) {
+        TagDto tag = tagService.getTag(id);
+        return ResponseEntity.ok(tag);
+    }
+
+    @DeleteMapping("/{id}")
+    public void deleteTag(@PathVariable Long id) {
```

Comment:
What HTTP response status code are you getting here? It's not obvious

Reply:
fixed